roles/confluent.test/molecule/plain-rhel/molecule.yml

```diff
@@ -100,6 +100,20 @@ provisioner:
       all:
         sasl_protocol: plain
 
+        # Testing secrets protection
+        secrets_protection_enabled: true
+        confluent_cli_download_enabled: true
```

Comment:
update the logic around confluent_cli_download_enabled variable, if i turn on secrets protection then the cli should get downloaded

Reply:
should be something like:
```
confluent_cli_download_enabled: "{{rbac_enabled or secrets_protection_enabled}}"
```
in the file roles/confluent.common/defaults/main.yml

Reply:
and therefore you do not need to set confluent_cli_download_enabled here